docs/03.API-EXAMPLE.md

```diff
@@ -551,6 +551,47 @@ Value of x is 12
 Value of x is 17
 ```
 
+## Changing the seed of pseudorandom generated numbers
+By default srand is not called in JerryScript, causing it to always generate the same random numbers every time it is being run.
+To solve this, you have to use srand in your program with a seed.
+
+```c
+#include <string.h>
+#include "jerryscript.h"
+
+int
+main (void)
+{
+  const jerry_char_t script[] = "var a = Math.random(); print(a);";
+  size_t script_size = strlen ((const char *) script);
+
+  bool ret_value = jerry_run_simple (script, script_size, JERRY_INIT_EMPTY);
+
+  return (ret_value ? 0 : 1);
+}
+```
+
+Running it would always output the same randomly generated numbers, as the seed is always the same.
+However, adding srand would use a different seed, therefore different random numbers are generated.
+
+```c
+#include <string.h>
+#include "jerryscript.h"
```

Comment:
Let's try prevent example code from bitrot, add `[doctest]: # ()` before the code block. This will guide the "doctest" testing phase to extract the example code and compile-link-run it (and will signal error if the code is not OK).